assets/create-template/templates/default/package.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "myTemplate",
+    "generator": {
+        "renderer": "react",
+        "supportedProtocols": [ ]
+    },
+    "dependencies": {
+        "@asyncapi/generator-react-sdk": "^0.2.25"
```

Comment:
I recommend to pick last version of generator-react-sdk 
https://github.com/asyncapi/generator-react-sdk/releases/tag/v1.0.20

Reply:
Done @Amzani